.github/actions/test-package/action.yml

```diff
@@ -25,6 +25,18 @@ inputs:
     description: 'Where to move the package.'
     required: false
     default: package.pkg
+  mvn-deploy:
+    description: 'Run mvn deploy.'
+    required: false
+    default: 'false'
+  mv-deploy-id:
+    description: 'Maven deploy id: runtime.verification or runtime.verification.snapshots'
+    required: false
+    default: 'runtime.verification.snapshots'
+  token:
+    description: 'GitHub token for publishing to GitHub Maven Packages.'
+    required: false
+    default: ''
 
```

Comment:
I would still not include it in the action, rather just in the overall release workflow. We know it only has to run once, so just one call to the mvn-deploy action should be enough, no need to make this action handle more things.

Reply:
Moving to release file.